crates/ruff_python_ast/src/symbol_imports.rs

```diff
@@ -0,0 +1,57 @@
+use std::fmt;
+
+/// A representation of an individual symbol imported via any import statement.
+pub enum AnyImport<'a> {
+    Import(Import<'a>),
+    ImportFrom(ImportFrom<'a>),
+}
+
+/// A representation of an individual symbol imported via an `import` statement.
+pub struct Import<'a> {
+    pub name: Alias<'a>,
+}
+
+/// A representation of an individual symbol imported via a `from ... import` statement.
+pub struct ImportFrom<'a> {
+    pub module: Option<&'a str>,
+    pub name: Alias<'a>,
+    pub level: Option<&'a usize>,
```

Comment:
Nit: `usize` is copy... and a `u32` should be sufficient to represent all valid python programs
```suggestion
    pub level: Option<u32>,
```

Reply:
It's a little painful to change to `u32` as long as RustPython's `ImportFrom` uses `usize` here :/